### DISCUSSION TITLE
Support for microsoft / Phi-3.5-vision-instruct

### DISCUSSION THREAD

**Discussion by Youho99** [microsoft/onnxruntime-genai#827] [Model support]
In the same idea of ​​support as phi-3-vision, and Florence-2 (see https://github.com/microsoft/onnxruntime-genai/discussions/658), integrate microsoft/Phi-3.5-vision-instruct which has just been released, in onnx in onnxruntime-genai.

There is a chance that the integration will be similar to that already done with phi-3-vision.

Thanks :)

**Comment by elephantpanda** [comment]
Yes, please. But also Phi 3 vision int4 (DirectML) isn't quite working yet so please don't forget about that one unless phi 3.5 vision solves those problems.

**Comment by kunal-vaishnavi** [comment]
The work is in progress to support Phi-3.5 vision. For example, [this PR](https://github.com/microsoft/onnxruntime-genai/pull/796) will add support for multiple images. We are also looking into the DirectML issues with Phi-3 vision.

**Comment by kunal-vaishnavi** [comment]
The new [Phi-3 vision](https://huggingface.co/microsoft/Phi-3-vision-128k-instruct-onnx) and [Phi-3.5 vision](https://huggingface.co/microsoft/Phi-3.5-vision-instruct-onnx) ONNX models have now been released. The new models support no-image, single-image, and multi-image scenarios.
